firefox_puppeteer/ui/windows.py

```diff
@@ -293,14 +300,15 @@ def get_property(self, property_id):
         """
         return self._l10n.get_property(self.properties, property_id)
 
-    def open_window(self, callback=None, expected_window_class=None):
+    def open_window(self, callback=None, expected_window_class=None, is_focused=True):
         """Opens a new top-level chrome window.
 
         :param callback: Optional, function to trigger the window to open. It is
          triggered with the current :class:`BaseWindow` as parameter.
          Defaults to `window.open()`.
-
         :param expected_class: Optional, check for the correct window class.
+        :param is_focused: Optional, waits until the new window will be focused.
+         Defaults to `True`.
```

Comment:
Oh, I changed the handling but missed to update the comment. This is just the expected focus state of the newly opened window. It can be `True` or `False`. So we would have to wait for both possible states. As you say opening windows in the background is rare, that's why I choose `True`. Would that make sense? I wouldn't explicitly expose the wait in the description given that we do it anyway.